slitlessutils/core/preprocess/crrej/group.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from slitlessutils.core.wfss import WFSSCollection
+from ....logger import LOGGER
+
+
+def group_by_visit(files, return_unique_visits=False, **kwargs):
+    """
+    Group a list of files based on their unique visits.
+
+    Parameters
+    ----------
+    files : list of str
+        List of filenames
+
+    return_unique_visits : bool, optional
+        Indicating whether to return unique visits along with grouped files.
+        If `True`, a tuple containing grouped files and unique visits is returned.
+        If `False`, only grouped files are returned. Default is `False`.
```

Comment:
```suggestion
        Indicates whether to return unique visits along with grouped files.
        If `True`, a list containing lists of grouped files and an array of unique
        visits are returned.
        If `False`, only grouped files are returned. Default is `False`.
```
I think this is more accurate to the data types returned.